editor/plugins/tiles/tile_set_atlas_source_editor.cpp

```diff
@@ -2555,9 +2590,17 @@ TileSetAtlasSourceEditor::TileSetAtlasSourceEditor() {
 	tool_advanced_menu_button->set_flat(true);
 	tool_advanced_menu_button->get_popup()->add_item(TTR("Create Tiles in Non-Transparent Texture Regions"), ADVANCED_AUTO_CREATE_TILES);
 	tool_advanced_menu_button->get_popup()->add_item(TTR("Remove Tiles in Fully Transparent Texture Regions"), ADVANCED_AUTO_REMOVE_TILES);
+	tool_advanced_menu_button->get_popup()->add_item(TTR("Remove Tiles Outside the Texture"), ADVANCED_CLEANUP_TILES);
 	tool_advanced_menu_button->get_popup()->connect("id_pressed", callable_mp(this, &TileSetAtlasSourceEditor::_menu_option));
 	tool_settings->add_child(tool_advanced_menu_button);
 
+	outside_tiles_warning_label = memnew(Label);
+	outside_tiles_warning_label->set_text(TTR("The current atlas source has tiles outside the texture.") + " (?)");
```

Comment:
That `(?)` is kind of a first in the editor UI language so far.

Might be worth starting a discussion with the new design team to see if we want to come up with a unified language for this. Shouldn't be blocking for this PR IMO, it can be refactored later.